src/Message/Response.php

```diff
@@ -618,6 +618,27 @@ public function getSummary()
         return null;
     }
 
+    /**
+     * Get the reason the subscription was canceled
+     *
+     * One of the reason codes documented here by Vindicia will be returned:
+     * https://www.vindicia.com/documents/2500ProgGuideHTML5/Default.htm#ProgGuide/Canceling_AutoBills\
+     * _with.htm%3FTocPath%3DCashBox2500ProgGuide%7C5%2520Working%2520with%2520AutoBills%7C5.3%2520\
```

Comment:
The link you provide is for v25 of the Cashbox API but the version we actually use is v18

Also you have this same link in several places and it will need to be updated each time with each version bump. We should find a single place for this link and you can reference it with `@see` or something similar.